dags/veda_data_pipeline/veda_process_vector_pipeline.py

```diff
@@ -99,8 +99,8 @@
         },
         network_configuration={
             "awsvpcConfiguration": {
-                "securityGroups": vector_ecs_conf.get("VECTOR_SECURITY_GROUP"),
-                "subnets": vector_ecs_conf.get("VECTOR_SUBNETS"),
+                    "securityGroups": mwaa_stack_conf.get("SECURITYGROUPS"),
```

Comment:
We are going to continue supporting dual VPCs (Vector + default). Suggest to continue using `VECTOR_SECURITY_GROUP`, `VECTOR_SUBNETS` and `VECTOR_VPC`

cc: @amarouane-ABDELHAK, @ranchodeluxe

Reply:
This commit should address the issue of optional vector sg and subnets: https://github.com/NASA-IMPACT/veda-data-airflow/commit/bad705e69f872bdd91fa225df2602da9efface73